includes/APIWrite.php

```diff
@@ -0,0 +1,258 @@
+<?php
+
+namespace MediaWiki\Extension\Example;
+
+use MediaWiki\Rest\SimpleHandler;
+use Wikimedia\ParamValidator\ParamValidator;
+use MediaWiki\MediaWikiServices;
+
+use Title;
+use WikitextContent;
+use WikiPage;
+
+# include / exclude for debugging
+error_reporting(E_ALL);
+ini_set("display_errors", 1);
+
+require_once("ApiUtil.php");
+require_once("Util.php");
+
+function selectToArray($db, $table, $col, $conds) {
+    $out = array();
+    $res = $db->select(
+        $table,
+        [$col],
+        $conds,
+    );
+    foreach ($res as $row) {
+        array_push($out, $row->{$col});
+    }
+    return $out;
+}
+
+// TODO move to Util.php
+function updateDomainManifest($witness_event_id, $db) {
```

Comment:
This probably needs to exist only on APIWrite. You need to remove the identical function in APIRead.

Reply:
Yes, makes sense.

Reply:
Done - [API_Split bb2b631] Remove unnecessary function updateDomainManifest from APIRead